src/components/App.tsx

```diff
@@ -2215,7 +2215,9 @@ class App extends React.Component<AppProps, AppState> {
       (shouldBind || parentCenterPosition)
     ) {
       container = getTextBindableContainerAtPosition(
-        this.scene.getElements().filter((ele) => !isTextElement(ele)),
+        this.scene
+          .getElements()
+          .filter((ele) => !isTextElement(ele) && !getBoundTextElement(ele)),
```

Comment:
Isn't `!isTextElement(ele)` check unnecessary here? Text elements can't be `ExcalidrawTextContainer`s